python/pyngraph/ops/softmax.cpp

```diff
@@ -24,10 +24,8 @@ namespace py = pybind11;
 
 void regclass_pyngraph_op_Softmax(py::module m)
 {
-    py::class_<ngraph::op::Softmax,
-               std::shared_ptr<ngraph::op::Softmax>,
-               ngraph::op::util::UnaryElementwiseArithmetic>
-        softmax(m, "Softmax");
+    py::class_<ngraph::op::Softmax, std::shared_ptr<ngraph::op::Softmax>, ngraph::op::Op> softmax(
+        m, "Softmax");
```

Comment:
Format is correct here ?

Reply:
I agree with you that previous version was more readable but clang forces such format ;)